DockerLaunchCommands.MD

```diff
@@ -0,0 +1,14 @@
+# Docker launch commands for coder
```

Comment:
You might want to fix the title cases of headings and subheads to match with the style of the README file. Also, I personally do no like upper-case file extensions (as ignore patterns are easier if extensions are kept lower-cased) and CamelCased file names, but that could just me.